src/main/resources/application.properties

```
@@ -0,0 +1,7 @@
+spring.application.name=delivery
+spring.data.mongodb.iri=mongodb://slava:12345@localhost:27017/delivery
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=delivery
+spring.data.mongodb.username=slava
+spring.data.mongodb.password=12345
```

```suggestion
spring.data.mongodb.password=12345

```

See how to configure your IDE https://github.com/ajax-tarnavskyi-m/BookExchangeService/pull/1#discussion_r1718917610

---

It's better to use yaml instead of properties